crm_exception/views/base_exception_views.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" ?>
+<odoo>
+     <record id="view_exception_rule_form_inherit" model="ir.ui.view">
+        <field name="name">exception.rule.form</field>
+        <field name="model">exception.rule</field>
+        <field name="inherit_id" ref="base_exception.view_exception_rule_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='model']" position="after">
+                <field
+                    name="stage_ids"
+                    widget="many2many_tags"
+                    attrs="{'invisible': [('model', '!=', 'crm.lead')]}"
+                    groups="base.group_system"
```

Comment:
Why assign this group here?

Reply:
I want to use `model` field in `attrs` and this field can be accessed by `base.group_system`.
https://github.com/OCA/server-tools/blob/97be5cf17cc32f4ef3e21fa493868426a5054b2d/base_exception/views/base_exception_view.xml#L44-L45

Reply:
@AungKoKoLin1997 We shouldn't need this, then, since the parent node already has the group.